ObjectPrintingTests/ObjectPrinterTests.cs

```diff
@@ -0,0 +1,210 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using FluentAssertions;
+using NUnit.Framework;
+using ObjectPrinting;
+
+namespace ObjectPrintingTests
+{
+    public class ObjectPrinterTests
+    {
+        private readonly string newLine = Environment.NewLine; 
+
+        [Test]
+        public void ForWithoutSettings_ShouldPrintAllPropertiesAndFields()
+        {
+            var pet = new Pet { Name = "Bob", Age = 5, Owner = null };
+            var printer = ObjectPrinter.For<Pet>();
+
+            printer.PrintToString(pet).Should()
+                .Be($"{nameof(Pet)}{newLine}" +
+                    $"\tAge = {pet.Age}{newLine}" +
+                    $"\tOwner = null{newLine}" +
+                    $"\tName = {pet.Name}{newLine}");
+        }
+
+        [Test]
+        public void ExcludingType_ShouldExcludeAllPropertiesAndFieldsWithType()
+        {
+            var pet = new Pet { Name = "Bob", Age = 5, Owner = null };
+            var printer = ObjectPrinter.For<Pet>()
+                .Excluding<Person>();
+
+            printer.PrintToString(pet).Should()
+                .Be($"{nameof(Pet)}{newLine}" +
+                    $"\tAge = {pet.Age}{newLine}" +
+                    $"\tName = {pet.Name}{newLine}");
+        }
+
+        [Test]
+        public void ExcludingProperty_ShouldExcludeOneProperty()
```

Comment:
```suggestion
        public void ExcludingProperty_ShouldExcludeThisProperty()
```
Чтобы было однороднее с остальными тестами)